config/locales/errors/en.yml

```diff
@@ -2,6 +2,7 @@ en:
   dictionary:
     invalid_email_error: &invalid_email_error "Enter an email address in the correct format, like name@example.com"
     blank_email_error: &blank_email_error "Enter an email address"
+    email_typo_error: &email_typo_error "The email address appears to have a typo"
```

Comment:
Some optional phrases:

- The email address appears to be mistakenly typed
- The email address seems to be mistaken
- The email address seems to be incorrect

Reply:
Yeah! I agree, `typo` sounds a bit slang.

I've just done some google searches (yeah I know, very scientific 🤣) and the combinations that gets more results are "seems to have" (as opposite to "appears to have") an "typing error" (as opposite to "typing mistake" or "mistakenly typed").

So I think we can go with: "The email address seems to have a typing mistake".

Reply:
Sounds good 👍